src/Symfony/Component/Form/Extension/Core/Type/CountryType.php

```diff
@@ -41,6 +42,10 @@ public function configureOptions(OptionsResolver $resolver)
     {
         $resolver->setDefaults(array(
             'choice_loader' => function (Options $options) {
+                if (!class_exists(Intl::class)) {
+                    throw new LogicException('The "symfony/intl" component is required to use the Country type.');
```

Comment:
I would prefer `LanguageType:class`, to get the FQCN here, same for the other exceptions

Reply:
Done.